pkg/maven/maven_test.go

```diff
@@ -190,3 +202,48 @@ func TestGetTestModulesExcludes(t *testing.T) {
 		assert.Equal(t, expected, modulesExcludes)
 	})
 }
+
+func TestMavenInstall(t *testing.T) {
+	t.Run("Should return path to jar file", func(t *testing.T) {
+		actual := jarFile("my-app")
+		assert.Equal(t, "target/my-app.jar", actual)
+	})
+
+	t.Run("Should return path to war file", func(t *testing.T) {
+		actual := warFile("my-app")
+		assert.Equal(t, "target/my-app.war", actual)
+	})
+
+	t.Run("Install a file", func(t *testing.T) {
+		execMockRunner := mock.ExecMockRunner{}
+		expectedParameters := []string{"--file", "pom.xml", "-Dfile=app.jar", "-DpomFile=pom.xml", "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn", "--batch-mode", "install:install-file"}
+
+		err := InstallFile("app.jar", "pom.xml", &execMockRunner)
+
+		assert.NoError(t, err)
+		assert.Equal(t, len(expectedParameters), len(execMockRunner.Calls[0].Params))
+		assert.Equal(t, mock.ExecCall{Exec: "mvn", Params: expectedParameters}, execMockRunner.Calls[0])
```

Comment:
This could crash the test runner, since execution continues.
```suggestion
		if assert.Equal(t, len(expectedParameters), len(execMockRunner.Calls[0].Params)) {
			assert.Equal(t, mock.ExecCall{Exec: "mvn", Params: expectedParameters}, execMockRunner.Calls[0])
		}
```